dockerfile

```diff
@@ -0,0 +1,28 @@
+
+FROM node:16 AS build-stage
```

Comment:
Please use `node:lts` instead node:16 to mitigate risks of compatibility in future

Reply:
Good suggestion!